src/commands/createFunctionApp/createCreateFunctionAppComponents.ts

```diff
@@ -54,7 +54,13 @@ export async function createCreateFunctionAppComponents(context: ICreateFunction
 
     await detectDockerfile(context);
 
-    promptSteps.push(new SiteNameStep(context.dockerfilePath ? "containerizedFunctionApp" : "functionApp"));
+    // SiteNameStep pre-requisites
+    if (!context.advancedCreation) {
+        LocationListStep.addStep(wizardContext, promptSteps);
+    } else {
+        CustomLocationListStep.addStep(wizardContext, promptSteps);
```

Comment:
Would appreciate a double check on this location logic to make sure these are the scenarios where each type of list step applies